# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoaderBase.h
# Commit: a80062ebc58c
# Full Hash: a80062ebc58cb7052b0cba86f7503b84cda49f9a
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-10-03 20:53 +0000
# Regressor Bug: 1968890
# File Overlap Count: 2
# Overlapping Files: js/loader/ModuleLoaderBase.h, js/loader/ModuleLoaderBase.cpp
# Description:
#   Bug 1968890 - Part 8: Remove FinishDynamicImportAndReject. r=jonco
# ==============================================================================

diff -r f0d873f64979 -r a80062ebc58c js/loader/ModuleLoaderBase.h
--- a/js/loader/ModuleLoaderBase.h	Fri Oct 03 20:53:29 2025 +0000
+++ b/js/loader/ModuleLoaderBase.h	Fri Oct 03 20:53:29 2025 +0000
@@ -553,20 +553,6 @@
                                              ModuleLoadRequest* aRequest,
                                              Handle<Value> aError);
 
-  /**
-   * Shorthand Wrapper for JSAPI FinishDynamicImport function for the reject
-   * case where we do not have `aEvaluationPromise`. As there is no evaluation
-   * Promise, JS::FinishDynamicImport will always reject.
-   *
-   * @param aRequest
-   *        The module load request for the dynamic module.
-   * @param aResult
-   *        The result of running ModuleEvaluate -- If this is successful, then
-   *        we can await the associated EvaluationPromise.
-   */
-  void FinishDynamicImportAndReject(ModuleLoadRequest* aRequest,
-                                    nsresult aResult);
-
   void RemoveDynamicImport(ModuleLoadRequest* aRequest);
 
   nsresult CreateModuleScript(ModuleLoadRequest* aRequest);
